src/player.cpp

```diff
@@ -2017,7 +2017,7 @@ void player::memorial( std::ostream &memorial_file, std::string epitaph )
     //HP
 
     const auto limb_hp = [this, &memorial_file, &indent]( const std::string & desc,
-                                                          const body_part bp ) {
+    const body_part bp ) {
```

Comment:
Try putting the start of the lambda on a new line.
```
   const auto limb_hp =
       [this, &memorial_file
```
Not sure that will fix it, just something to try.
Astyle doesn't understand lambdas very well.